hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -715,14 +718,54 @@ ppResult name (Report tests discards result) =
         ppDiscardCount discards <>
         ", passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
 
     OK ->
       pure . icon SuccessIcon '✓' . WL.annotate SuccessHeader $
         ppName name <+>
         "passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
+
+ppClassification :: Classification -> Doc Markup
+ppClassification (Classification classifiers total) =
+  if Map.null classifiers then
+    mempty
+  else
+    (<>) WL.linebreak $ WL.indent 4 . WL.align . WL.vsep $
+      (\(ClassifierName k, v) -> WL.text $ show (occurrenceRate v total) <> "% " <> k) <$> Map.toList classifiers
+
+occurrenceRate :: Classifier -> Integer -> Double
+occurrenceRate (Classifier _ occurrences) total =
+  let
+    percentage :: Double
+    percentage = fromIntegral occurrences / fromIntegral (total - 2) * 100
+    thousandths :: Integer
+    thousandths = round $ percentage * 10
+  in fromIntegral thousandths / 10
```

Comment:
Sorry, this should be:
```haskell
  let
    percentage :: Double
    percentage =
      fromIntegral occurrences / fromIntegral (total - 2) * 100
    thousandths :: Integer
    thousandths =
      round $ percentage * 10
  in
    fromIntegral thousandths / 10
```

The basic rule is to try and keep everything aligned to a multiple of two spaces, so we always wrap after `=` and `in`